packages/server/test/unit/config_spec.coffee

```diff
@@ -3,6 +3,7 @@ require("../spec_helper")
 _        = require("lodash")
```

Comment:
Is there an existing e2e test/example project where we could add a true end-to-end test for this, somehow? This is such a weird bug that I don't know if only unit tests should suffice.